packages/itwinui-react/src/core/Table/TableExpandableContentMemoized.tsx

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import * as React from 'react';
+import { Box } from '../../utils/index.js';
+import type { PolymorphicForwardRefComponent } from '../../utils/index.js';
+
+type TableExpandableContentProps = {
+  isDisabled: boolean;
+  children: React.ReactNode;
+  isSelected?: boolean;
+};
+
+const TableExpandableContent = React.forwardRef((props, ref) => {
+  const { isDisabled, children, isSelected, className, style, ...rest } = props;
+  return (
+    <Box
+      className={cx('iui-table-row', 'iui-table-expanded-content', className)}
+      style={{
+        flex: '0 0 auto',
+        minWidth: '100%',
+        ...style,
+      }}
+      aria-disabled={isDisabled}
+      aria-selected={isSelected}
```

Comment:
Good catch. Neither of these ARIA attributes are being used correctly, because the expandable-content is a generic `<div>`. I think we only use `aria-disabled` in our CSS (which we should probably change to a data attribute). The selection state is handled using the `+` selector.

https://github.com/iTwin/iTwinUI/blob/55377685ee5f2f361ae762a840e7e72c321d5026/packages/itwinui-css/src/table/base.scss#L241

Reply:
Should I remove this aria attribute in `TableRowMemoized` as well since this is also a generic `<div>`?

Reply:
No, we rely on `aria-selected` to style `TableRow`. Let's make a note to look into changing these to data attributes in the future.

https://github.com/iTwin/iTwinUI/blob/55377685ee5f2f361ae762a840e7e72c321d5026/packages/itwinui-css/src/table/base.scss#L220

Reply:
I have this as an after-PR todo and removed the aria attributes in the expandable content.

Reply:
Yes, we never suggested removing `aria-disabled`, only `aria-selected` (because it wasn't present before this PR). Please re-read the thread and feel free to ask any questions. The changes in your latest commit look fine, but just make sure to revert the playground.

Reply:
I have re-read and everything is clear to me. Merged the PR.